spec/support/service_catalog/service_offering.rb

```diff
@@ -0,0 +1,51 @@
+module ServiceCatalog
```

Comment:
@syncrou 
What is this file, is it a spec or a support file. We already have a spec file in 
https://github.com/ManageIQ/service_portal-api/blob/master/spec/services/service_catalog/service_offering_spec.rb

Reply:
Support file

Same as my comment above.  

Any spec calling `create` on `PortfolioItem` was causing me lots of pain. This allows those requests to work with real data see line 17 here: https://github.com/ManageIQ/service_portal-api/pull/82/files#diff-da5419dcb98a21e5282dcb098c832fea